### DISCUSSION TITLE
Linux Osu!Lazer wont start with no errors or logs

### DISCUSSION THREAD

**Discussion by SollyBunny** [ppy/osu#15720] [Q&A]
Osu ran fine before updating (I updated when the update notification appeared) and now wont start giving no error msgs/output or any logs in ~/.local/share/osu/logs
Tried running it directly in bin and with the run script

**Comment by gagahpangeran** [comment]
Is there no output when you launch the AppImage from terminal?

**Reply by SollyBunny** [reply]
none at all

**Reply by bdach** [reply]
please check `dmesg` for any error messages.

**Comment by DarioTD** [comment]
I used AUR "osu-lazer-bin" to update it and it logs
